src/tests/ftest/io/dfuse_bash_cmd.yaml

```diff
@@ -1,8 +1,6 @@
 hosts:
   test_servers:
     - server-A
```

Comment:
Thanks Saurabh. Will the test still pass since it is passing a hostlist while there will be no client?

Reply:
> Thanks Saurabh. Will the test still pass since it is passing a hostlist while there will be no client?

Yeah, it will pass. Now for dfuse tests, if there is no client hostlist specified, the test runner node will be used as the host node.
Here is the link for passed dfuse_bash_cmd.py test - https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6683/1/testReport/FTEST_io/
For some reason, this PR got Node provisioning failure for Functional HW Large tests, so I need to re-trigger it.